src/state/slices/opportunitiesSlice/resolvers/uniV2/index.ts

```diff
@@ -123,13 +121,13 @@ export const uniV2LpOpportunitiesMetadataResolver = async ({
         }
       }
 
-      token0Decimals = zapperAppBalanceData.tokens[0].decimals
-      token1Decimals = zapperAppBalanceData.tokens[1].decimals
-      token0Reserves = bnOrZero(zapperAppBalanceData.dataProps.reserves[0])
-      token1Reserves = bnOrZero(zapperAppBalanceData.dataProps.reserves[1])
-      token0Address = ethers.utils.getAddress(zapperAppBalanceData.tokens[0].address)
-      token1Address = ethers.utils.getAddress(zapperAppBalanceData.tokens[1].address)
-      apr = bnOrZero(zapperAppBalanceData.dataProps.apy).toFixed()
+      token0Decimals = zapperAppBalanceData?.tokens?.[0].decimals!
```

Comment:
feel like this will blow up on us

Reply:
9c3049f80a0c87e4de5094e359e857d412a66e11

Reply:
Note we still need the non-null assertion because ts is a bit derpy here